gravatar-ui/src/main/java/com/gravatar/ui/components/atomic/Avatar.kt

```diff
@@ -144,11 +151,70 @@ private fun EmptyAvatar(size: Dp, modifier: Modifier = Modifier) {
 private fun Avatar(model: Any?, size: Dp, modifier: Modifier) {
     AsyncImage(
         model = model,
-        contentDescription = "User profile image",
```

Comment:
This was hardcoded in the component, we could probably skip the waiting for the translation for this one, as it will delay the release again.